### ISSUE TITLE
Collaboration Cycle for [Team Name, Product Name, Feature Name]

### ISSUE THREAD

**Issue by tameracorson** [department-of-veterans-affairs/va.gov-team#44401]
### VFS team name

MOVE

### Product name

1095-B

### Feature name

1095-B 

### GitHub label for product

1095b-tax-form

### When did/will you start working on this product/feature?

January 2022

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Both

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

No

### Do you need to capture any additional analytics or metrics?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/1095b-tax-form/product/1095B-Tax-Form-Product-Outline.md

### Notify the Collaboration Cycle team of this ticket in the vfs-platform-support Slack channel.

- [X] I acknowledge that I must notify \#vfs-platform-support after submitting this issue.

**Comment by tameracorson** [comment]
Hi @chaseakirby Just following up for next steps.  Shira told us to drop a new kickoff ticket and you would give direction.  I have made comment and closed staging ticket on MOVE 1095-B MVP due to IRS form inaccessibility, the team went back to drawing board for new design and build for1095-B react widget.   @CarlDickerson Will any content on the static page change?

**Comment by CarlDickerson** [comment]
@tameracorson @chaseakirby no content will be changed on the unauthenticated (not signed in) static page. The changes made only apply to the authenticated experience once users are signed in.

**Comment by tameracorson** [comment]
**Contact center review ticket here** : https://app.zenhub.com/workspaces/1095b-tax-form-61e19735167b2500170b2146/issues/department-of-veterans-affairs/va.gov-team/44593

**Comment by tameracorson** [comment]
**Analytics Implementation ticket here** : https://app.zenhub.com/workspaces/1095b-tax-form-61e19735167b2500170b2146/issues/department-of-veterans-affairs/va.gov-team/44810

**Comment by tameracorson** [comment]
**Privacy, security and infrastructure ticket here**: https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/issues/544

**Comment by tameracorson** [comment]
**Release plan here**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/1095b-tax-form/product/1095b-release-plan.md

**Comment by tameracorson** [comment]
**MVP Designs here**: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/1095b-tax-form/product/Business%20Owner%20Brief%2029JUL/1095-B%20MVP%20Product%20Designs.pdf

**Comment by tameracorson** [comment]
**MVP Prototype with hotpots enabled here**: https://www.sketch.com/s/1f2a2fbd-8d48-47c6-8e95-b26d5259a211/prototype/a/2F852299-DA7F-44DC-82E2-3B5CEC399F05

**Comment by tameracorson** [comment]
Hi @RLHecht just tagging you to our staging ticket so your team can see any staging review feedback for 1095-B that may impact Drupal static page or how our react widget / toggle is interacting with it..   For Timing (projected) :  **Staging review** 18 Aug @1430 ET, **Dark launch** 22 Aug with friendly vet testing.  **Moderated UAT** 25-26 Aug.   **Staged roll out** Canary test 25% 26-30 August; stage B 50% 30 Aug-4 September, Go live 5 Sept.

**Comment by tameracorson** [comment]
**URL**: http://preview-prod.vfs.va.gov/preview?nodeId=44346

**Comment by tameracorson** [comment]
@khandelwal doing QA of ticket.  Is there anything we need to do with this section:

Staging URL and staging test user information:
Note: Please double-check that you've provided staging access information appropriate for testing the tool or feature. Don't put staging credentials in your va.gov-team ticket; store test user information, passwords, and tasks in a .md file in the va.gov-team-sensitive repository.

**Comment by DanSmithSC** [comment]
![Screen Shot 2022-08-12 at 3.08.45 PM.png](https://images.zenhubusercontent.com/62e91ea65e4933fcf47cc6c0/866ab9cb-ee6d-48b1-8856-2d7ceb1ca66e)

Test Rail Test Run - Covering Required Staging Accessibility Review Tasks and Basic Test Plan/Test Cases. 

Should Cover the Required QA Artifacts inside of TestRail:
https://dsvavsp.testrail.io/index.php?/runs/view/3511&group_by=cases:section_id&group_order=asc

**Comment by briandeconinck** [comment]
@tameracorson Checking in ahead of the Staging Review this week -- I see the [preview link you shared](http://preview-prod.vfs.va.gov/preview?nodeId=44346), are there user credentials you're able to provide that will work with the preview link? I know testing authenticated views in Drupal previews has been an issue in the past, hoping you've figured it out!

**Comment by tameracorson** [comment]
@briandeconinck Great question and think you for starting our review.  According to our meeting w/ Shira, Drupal is an ongoing issue that has not been resolved.   I'm including my lead tech @khandelwal and developer @stephte and they may be able to help you.

**Comment by briandeconinck** [comment]
@tameracorson Okay, thank you! cc @allison0034 @chaseakirby for awareness.

If it's not possible to do a hands-on test with the authenticated view, could you provide a link to the React component in GitHub? A code review isn't as good as live testing with assistive technology, but since the product is pretty straightforward I think it should probably be fine for accessibility testing.

**Comment by allison0034** [comment]
Thank you! @tameracorson  Design and content can test with screen shots too if that helps.

**Comment by khandelwal** [comment]
Yes, we can provide a link to the React component in Github. As well as screenshots, video walkthrough and clickable prototype. We're not excited about not being able to test either, so happy to support any efforts to check!

**Comment by khandelwal** [comment]
@briandeconinck Is this what you need: https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/static-pages/download-1095b 

Do you also need the link to where the data loading scripts etc are (in the vets-api repo)?

**Comment by briandeconinck** [comment]
@khandelwal I think that should be all I need! Thanks!

**Comment by allison0034** [comment]
@tameracorson I see you added a prototype and design comps in the comments above. Can you please add the correct design we need to review in the staging artifacts section of the ticket? The prototype and design PDF are different.

**Comment by tameracorson** [comment]
@allison0034 resurfaced from above.  Please advise if you need anything else.  Linking @CarlDickerson for visibility. 
**Link to Authenticated Experience screenshots**:

MVP Prototype with hotpots enabled here: https://www.sketch.com/s/1f2a2fbd-8d48-47c6-8e95-b26d5259a211/prototype/a/2BA4EE6C-B179-45B6-B951-499B96099C49 

MVP Designs here: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/1095b-tax-form/product/Business%20Owner%20Brief%2029JUL/1095-B%20MVP%20Product%20Designs.pdf 

Static Drupal page : http://preview-prod.vfs.va.gov/preview?nodeId=44346

**Comment by allison0034** [comment]
Thank you @tameracorson! Not everyone has Sketch, but you can link all 3 if you want. We just need a way to access any screens that will be changing. I know the Sketch file posted 5 days ago was originally different than the PDF

**Comment by tameracorson** [comment]
@CarlDickerson Can you drop the Sketch file for Unauthenticated, Authenticated, error states here? and I will move them up in the ticket to the right place.

**Comment by CarlDickerson** [comment]
> Thank you @tameracorson! Not everyone has Sketch, but you can link all 3 if you want. We just need a way to access any screens that will be changing. I know the Sketch file posted 5 days ago was originally different than the PDF

@allison0034 Attached is PDF version of designs
[1095-B_MVP_Designs_081622.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/9353132/1095-B_MVP_Designs_081622.pdf)
.
